rfcs/0023-glossary-terms.md

```diff
@@ -101,6 +92,41 @@ The terms proposed are below:
 - Trustless nodes - again, some lifting required.
 - Light node / light client - has a negative connotation of not being able to validate chain state.
 
+### User Transaction
+**Concept:** A transaction issued by a user - currently a payment or a delegation change.
+
+**Rationale:** This term clearly describes the concept, and leaves room for further types of user issued transactions.
+
+**Downsides:** This is a subset of "Transactions", so it is a bit redundant.
+
+**Usage:** There are three types of transactions in the Coda network currently - user transactions, fee transfers, and coinbases. User transactions allow users to manage accounts and send money.
+
+**Alternatives:**
+- User command - this term is confusing, especially in  docs, as it can be conflated with a CLI command that a user issues.
+
+### Protocol Transaction
+**Concept:** A transaction issued by the protocol - currently a fee transfer or coinbase (both are structurally identical).
+
+**Rationale:** In discussions with parties that were not familiar with the protocol, fee transfers were misunderstood as representations of fees associated with a user transaction. As soon as it was explained that these were transactions programatically issued by the protocol rather than a user, it became clear. As such, it is recommend to partition transactions into *user transactions* and *protcol transactions*.
```

Comment:
```suggestion
**Rationale:** In discussions with parties that were not familiar with the protocol, fee transfers were misunderstood as representations of fees associated with a user transaction. As soon as it was explained that these were transactions programatically issued by the protocol rather than a user, it became clear. As such, it is recommend to partition transactions into *user transactions* and *protocol transactions*.
```